src/include/daos/object.h

```diff
@@ -94,6 +94,18 @@ daos_ec_cs_valid(uint32_t cell_sz)
 	return true;
 }
 
+static inline bool
+daos_ec_pda_valid(uint32_t ec_pda)
+{
+	return ec_pda > 0;
+}
+
+static inline bool
+daos_rp_pda_valid(uint32_t rp_pda)
+{
+	return rp_pda > 0;
+}
```

Comment:
I don't know a lot about this feature, but I wonder if these checks are thorough enough. Is there a maximum value over which behavior is undefined or otherwise doesn't make sense?

Reply:
Yup, this is good question, @gnailzenh Any idea for this?